tests/python_client/common/common_func.py

```diff
@@ -431,6 +431,34 @@ def gen_search_param(index_type, metric_type="L2"):
     return search_params
 
 
+def gen_search_pagination_param(index_type, offset=10, metric_type="L2"):
```

Comment:
why write a new function rather than modifying the old function "gen_search_param"

Reply:
> why write a new function rather than modifying the old function "gen_search_param"

Good question. I didn't know how to modify it at first, but I can now.